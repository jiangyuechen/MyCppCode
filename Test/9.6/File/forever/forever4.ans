334 125772154
