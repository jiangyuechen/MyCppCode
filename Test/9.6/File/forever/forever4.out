16816 677563268
